src/pages/index.js

```diff
@@ -236,6 +236,20 @@ const IndexPage = ({ location }) => {
               </li>
             </ul>
           </Card>
+          <Card>
+            <h3>Platform automation</h3>
+            <ul>
+              <li>
+              <Link to={"/platform-automation/"}>Platform automation</Link>  
+              </li>
+              <li>
+              <Link to={"/alertmanager/"}>AlertManager</Link>
+              </li>
+              <li>
+               <Link to={"/automated-scaling/"}>Automated scale-down</Link> 
+              </li>
```

Comment:
Should these be re-ordered to match the `sort_order` in the markdown files?

Reply:
The sort order in the .md files on the left navigation are determined based on the number inside the files under the content_owner section. 

For the order in the homepage section, in index.js - we determine the order in which they are placed.

Reply:
I agree that it makes sense to match the `sort_order` and the order listed in the homepage defined on index.js